.editorconfig

```diff
@@ -0,0 +1,15 @@
+# EditorConfig is awesome: http://EditorConfig.org
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
+
+[*.{html,css,js}]
+charset = utf-8
+indent_style = space
+indent_size = 2
+
+[*.go]
+indent_style = tab
+indent_size = 8
```

Comment:
For `{html,css,js}`? If so, I really don't care. I just like that there's codeguide.co which one can point to if some code isn't properly formatted. Beyond just indentations.